README.md

```diff
@@ -24,6 +24,19 @@ Used API [Yandex.Speller](https://tech.yandex.ru/speller/doc/dg/concepts/About-d
 ## Using CLI
 `yaspeller [options] <file-or-directory-or-link...>`
 
+## Using with pre-commit:
+
+Add this to your `.pre-commit-config.yaml`:
+
+```yaml
+- repo: https://github.com/hcodes/yaspeller.git
+  rev: master # Use the sha / tag you want to point at or master
```

Comment:
the default configuration should not suggest master -- [this has more details](https://pre-commit.com/#using-the-latest-version-for-a-repository) -- generally I usually put either some fake value that's invalid here (such as `''`) or the latest tag